File: Prefix Sum/Easy/1732. Find the Highest Altitude/find_the_highest_altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int curAltitude = 0;
        int highestPoint = 0;
        for (int g:gain)
        {
            curAltitude += g;
            highestPoint = max(highestPoint, curAltitude);
        }
        return highestPoint;
    }
};

File: Prefix Sum/Easy/1732. Find the Highest Altitude/find_the_highest_altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        curAltitude = 0
        highestPoint = 0
        for g in gain:
            curAltitude += g
            highestPoint = max(highestPoint, curAltitude)
        return highestPoint
